exporter/datadogexporter/internal/metadata/metadata.go

```diff
@@ -59,6 +60,13 @@ type HostMetadata struct {
 
 	// Tags includes the host tags
 	Tags *HostTags `json:"host-tags"`
+
+	//Gohai payload contains inventory of system information
+	// this is embedded because of special serialization requirements
+	gohai.Payload
+
+	// Backend treats processes as resources
```

Comment:
In Go, documentation has to start with the name of the identifier: https://go.dev/blog/godoc
```suggestion
	// Processes specifies a payload which ...
```
What does it contain?